docs/source/user_guide/tutorials/getting_started.rst

```diff
@@ -253,484 +133,518 @@ For example, if you ran the ``easylink run`` command on your laptop, the impleme
+
+Finally, since we are using simulated input datasets, and therefore know the ground truth of 
+which records are truly links, we can directly see how our naive model performed with the help of 
+a script to evaluate false positives and false negatives, :download:`print_fp_fn_w2_ssa.py`.
+Download and run it::
+
+  $ python print_fp_fn_w2_ssa.py results/2025_06_26_10_13_31 .996
+  9292 true links
+  For threshold 0.996, len(false_positives)=19; len(false_negatives)=188
+
+In other words, with a threshold 
+probability of 99.6%, out of 9,262 true links to be found, our model missed 19 (false negatives),
+and additionally linked 188 pairs that shouldn't have been linked (false positives). 
+
+
+Depending on our goals with the linked data, we might increase the threshold to reduce false positives,
+at the cost of increased false negatives.
+But this was a simple linkage model.
+Let's improve it to see if we can get a better performance tradeoff!
+
+
+Configuring an improved pipeline
+================================
+Next, let's modify our naive pipeline configuration YAML to try to improve our results. Primarily, we 
+will change the ``COMPARISONS`` we pass to ``splink_evaluating_pairs`` to use flexible comparison 
+methods rather than exact matches, allowing us to link records which have typos or other noise in them. We'll 
+use a new pipeline configuration YAML, :download:`pipeline_demo_improved.yaml`, with these changes.
+
+In ``splink_evaluating_pairs``, we make the following changes:
+
+.. code-block:: diff
+
+     LINK_ONLY: true
+     BLOCKING_RULES_FOR_TRAINING: "'l.first_name == r.first_name,l.last_name == r.last_name'"
+  -  COMPARISONS: "ssn:exact,first_name:exact,middle_initial:exact,last_name:exact"
+  +  COMPARISONS: "ssn:levenshtein,first_name:name,middle_initial:exact,last_name:name"
+     PROBABILITY_TWO_RANDOM_RECORDS_MATCH: 0.0001  # == 1 / len(w2)
+
+
+
+``COMPARISONS`` now uses 
+`Levenshtein <https://moj-analytical-services.github.io/splink/api_docs/comparison_library.html#splink.comparison_library.LevenshteinAtThresholds>`_
+comparisons for ``ssn``, and 
+`Name <https://moj-analytical-services.github.io/splink/api_docs/comparison_library.html#splink.comparison_library.NameComparison>`_
+comparisons for ``first_name`` and ``last_name``, to link similar but not identical SSNs and names.
+
+By re-running the pipeline with these changes and then running the evauation script, we can see how our results compare::
+
+  $ easylink run -p pipeline_demo_improved.yaml -i input_data_demo.yaml -e environment_local.yaml -I /mnt/team/simulation_science/priv/engineering/er_ecosystem/images
+  $ python print_fp_fn_w2_ssa.py results/2025_06_26_11_08_57 .996
+  9292 true links
+  For threshold 0.996, len(false_positives)=19; len(false_negatives)=158
+
+We eliminated 30 false negatives compared to the naive results, thanks to our model linking more records with columns that 
+are similar but don't exactly match.
```

Comment:
Maybe also say that the number of false positives, 19, stayed the same, so using improved comparisons found us more links without giving us more bad links, which is better than simply reducing the match threshold.